### PR TITLE
`General`: Show line breaks in manual feedback

### LINKED ISSUE #3035: Line breaks in the additional feedback are lost
### Describe the bug
When giving feedback to a text exercise with the "Additional feedback" feature below the student's submission, it is possible to enter text with line breaks. This can be used to enter detailed feedback with multiple bullet points. However, when the student sees the feedback, the line breaks have been removed and the bullet points have been squashed into an unstructured blob of text.

This seems to be an issue with the display of the feedback and not with storing the feedback. When going back to the correction of the exercise, all the line breaks are still there.

#### To Reproduce
1. Have a submission to a text exercise
2. Give feedback to it, using the "Additional feedback" feature. This feedback should contain line breaks.
3. Review the feedback from the student's perspective and see, that the line breaks are gone.

#### Expected behavior
The line breaks should not be removed. For example, programming exercises show the expected behavior in this regard.

#### Screenshots
This is the correction view:
grafik

This is the student view:
grafik

### Environment
Artemis 4.10.6

### LINKED ISSUE #3854: MultiLine Support for Feedback view of Students
### Is your feature request related to a problem?

Since we format our feedback using multiple lines, the feedback should be shown to the students in multiple lines as well ..

### Describe the solution you'd like

Format Feedback with multiple lines if written that way ..

Student View (**that shall show multiple lines as well**):
image

Reviewer/Assessment View:

image

Detailed Feedback View (Student):
image

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by b-fein**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-tests/).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context
Closes #3035
Closes #3854 
Closes kit-sdq/Artemis#15

### Description
Manual comments by tutors could contain multiple lines. Those were not always shown to the student, however. This PR makes sure that the newlines are always kept when displaying the feedback to students.

The first line of the feedback is in the same line as the ‘Feedback’ text. I think this only looks a bit odd because I used very short lines in the screenshots. In practice and in my experience I think that most tutors will give a longer/full-sentence explanation in the first line and only then add some more lines with additional context/bullet-points. Therefore, I think keeping the first line of the feedback text on the same line as ‘Feedback’ is fine.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise with manual feedbacks
- 1 Text Exercise with manual feedbacks

1. Submit as the student to the programming exercise and the text exercise.
2. As instructor, create manual feedbacks (both inline and additional ones) containing one or multiple lines for those submissions.
3. Log in as student again and look at your results. The feedback should still contain multiple lines. 

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Test Coverage
unchanged; relevant method in `feedback.model.ts`: 2x 100%

### Screenshots
Additional feedback:
Screenshot 2022-02-18 at 14-37-39 Code Editor

Inline code feedback:
Screenshot 2022-02-18 at 14-38-25 Code Editor

Inline text feedback:
Screenshot 2022-02-18 at 14-37-56 Text Exercises

Modeling exercises:
Screenshot 2022-02-18 at 14-30-22 Modelling Exercises

### CODE REVIEW

File: src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts [hunk 1]
```diff
@@ -128,11 +128,10 @@ export class CodeEditorTutorAssessmentInlineFeedbackComponent {
...
+        } else if (feedback.detailText) {
+            feedbackText = feedback.detailText;
         }
-        if (feedback.detailText) {
-            return feedback.detailText;
-        }
-        return feedbackText;
+
+        return convertToHtmlLinebreaks(feedbackText);
     }
```
Comment (willergeorg) [inline src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts:136]: This method is quite similar to the one in `feedback.model.ts`. Do you see an uncomplicated way to refactor the logic in order to implement it once and reuse it at the two spots?

**Review by willergeorg — COMMENTED** [review]
Tested on TS1, worked as expected. I left one comment that is just a suggestion/question regarding code re-usability.

File: src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts (hunk 1 above)
Comment (b-fein) [reply src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts]: Yes, after I had another look it was quite simple. I implemented it in a281e067aaa725fdc4c55a922d0e4325eb630f08.

**Review by Strohgelaender — APPROVED** [review]
Tested on TS1, works as expected.

**Review by VinnavanSembian — APPROVED** [review]
Tested on ts1 with @lBruchmann and @Strohgelaender. Everything works well!

**Review by lBruchmann — DISMISSED** [review]
Tested ts1. Tested together with @VinnavanSembian and @Strohgelaender. Everything works as expected. Line breaks are correctly displayed.

**Review by dveljacic — APPROVED** [review]
Love this fix. And it seems to be working well on TS2. ✅

**Review by willergeorg — COMMENTED** [review]
There is one unused import in [code-editor-tutor-assessment-inline-feedback.component.ts](https://github.com/ls1intum/Artemis/pull/4717/files#diff-91aeebcad06f89c374a1d2fa8384af69b977bb1f990a99b246269d54a62d7464). Besides that, everything is fine for me.

**Review by ole-ve — APPROVED** [review]
Code looks good to me 👍

**Review by MertAksehirlioglu — APPROVED** [review]
Tested on TS1 and it works as expected.

**Review by Bofan-Zhu — APPROVED** [review]
Works as exepected on ts2 👍

**Pull request merged**